### PR TITLE
Fix flake errors, assuming line length of 120 chars

### PR DESCRIPTION
**Pull request by mike7c2**
This is a purely cosmetic set of changes to help flake8 to be quiet. 

In some environments flake8 is run through the CI system blocking PRs which have flake errors in them - this may save some others a little pain.

The changes were made assuming a 120 line length which was inferred from the code

### CODE REVIEW

**Comment by pavel-kirienko** [comment]
This is cool, thanks. Maybe we should also drop a few lines into README explaining how to use this thing?

**Comment by spiremike** [comment]
I have added another commit with some info in the `README.md`. I ended up adding a `setup.cfg` for flake8 too so that people don't have to configure it themselves for the project. it's possible in the future the project will decide some errors are not important and this gives us a way to do this for everyone running flake.

File: README.md [hunk 1]
```diff
@@ -89,3 +89,15 @@ tar czvf libcanard.tgz cov-int
 ```
 
 Then upload the resulting archive to Coverity.
+
+### Running flake8 for python code
+
+Flake8 is a tool for checking correctness and style of python code, [Flake 8 docs](http://flake8.pycqa.org/en/latest/).
```
Comment (pavel-kirienko) [inline README.md]: If this is a link, please make it read as a link. For example:

> [Flake8](http://flake8.pycqa.org/en/latest/) is a tool for checking correctness and style of python code.

Or:

> Flake8 is a tool for checking correctness and style of python code; the docs are available at <http://flake8.pycqa.org/en/latest/>.

File: README.md [hunk 2]
```diff
@@ -89,3 +89,15 @@ tar czvf libcanard.tgz cov-int
...
+
+The setup for Flake8 is defined in `setup.cfg`.
+
+Flake8 should be run from the top level directory, ideally code should be reviewed to make sure there are no
+flake errors before it is merged.
+```
+libcanard$ flake8
```
Comment (pavel-kirienko) [inline README.md]: The command doesn't seem to make sense. Did you mean that `flake8` needs to be run from the root directory? perhaps it's best to spell it out?

**Pull request merged**